share/clus_apis.go

```diff
@@ -669,6 +669,10 @@ func CLUSSigstoreTimestampKey() string {
 	return fmt.Sprintf("%s%s", CLUSConfigStore, "sigstore_timestamp")
 }
 
+func CLUSRemoteRepositoryKey(nickname string) string {
```

Comment:
Why not put the settings in object/config/system?
Separate it from object/config/system will require changes on configmap & kv backup/restore about the new key object/config/remote_export_repository but I don't see it in this PR

Reply:
Implemented.